### PR TITLE
Please add Dithereum Mainnet on Chain ID 24

### PR DESCRIPTION
**Pull request by solidityx**
This is fresh PR.. and will delete the old one. Thank you.

### CODE REVIEW

**Comment by solidityx** [comment]
In this comment, we have icon file..
https://github.com/ethereum-lists/chains/blob/master/_data/icons/dithereum.json

hope its all good now :)

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `dth` more than once

**Comment by solidityx** [comment]
I really appreciate you spending time on this.. and help me to perfect this.. 

I changed the short name.. thank you

File: _data/chains/eip155-24.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "nativeCurrency": {
+    "name": "Dither",
+    "symbol": "DTH",
+    "decimals": 18
+  },
+  "infoURL": "https://dithereum.org",
+  "shortName": "dthmainnet",
+  "chainId": 24,
+  "networkId": 24,
+  "ens": {}
```
Comment (ligi) [inline _data/chains/eip155-24.json:21]:
```suggestion
  "networkId": 24
```

**Pull request merged**